core/shared/src/test/scala/cats/parse/Rfc5234Test.scala

```diff
@@ -107,15 +107,15 @@ class Rfc5234Test extends munit.ScalaCheckSuite {
       )
       .map(_.mkString)
     forAll(genLwsp) { (s: String) =>
-      assertEquals(Rfc5234.lwsp.parseAll(s), Right(()))
+      assertEquals(Rfc5234.lwsp.parseAll0(s), Right(()))
```

Comment:
can we just call parseAll here, I think by the time you are parsing, you don't really care about the distinction.